app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+    for elem in phrase:
+        if elem.lower() == letter.lower():
+            count += 1
+    return count
```

Comment:
Try to write 1-line solution

Reply:
hello!
i fix my solution in 1-line solution 👍 

return phrase.lower().count(letter.lower())